docker/relay/cf-execd.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM centos:8
+
+RUN \
+        ln -sf /bin/true /usr/sbin/service && \
+        ln -sf /bin/true /bin/systemctl && \
+        ln -sf /bin/true /usr/bin/systemctl
+
+RUN \
+        yum -y install diffutils curl && \
+        curl -o rudder-setup https://repository.rudder.io/tools/rudder-setup && \
+        sh rudder-setup add-repository 6.1 && \
+        yum -y install rudder-agent && \
```

Comment:
no need to install the agent + relay, as the agent is already a relay dependency, so relay is enough